docs/source/index.rst

```diff
@@ -44,23 +44,16 @@ openwisp-radius
    :maxdepth: 2
    :caption: Contents:
 
-   /general/setup
-   /general/settings
-   /general/freeradius
-   /general/management_commands
-   /general/importing_users
-   /general/generating_users
-   /general/enforcing_limits
-   /general/registration
-   /general/social_login
-   /general/api
-   /general/how_to_extend
-   /general/contributing
+   /developer/setup
+   /user/settings
+   /developer/freeradius
+   /user/management_commands
+   /user/importing_users
+   /user/generating_users
+   /user/enforcing_limits
+   /user/registration
+   /user/social_login
+   /user/api
+   /developer/how_to_extend
+   /developer/contributing
    /general/goals
-
-Indices and tables
-==================
-
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
```

Comment:
I don't know why these links are here, but they are not setup and 2 of them output empty pages and the `modindex` returns 404, check here: https://django-freeradius.readthedocs.io/en/latest/ ,please correct if I am wrong and these are required.

Reply:
maybe they're just leftovers, I don't know and don't care so much now, good catch, let's simplify.